students/AntonBakaiev/hw-4/answer.js

```diff
@@ -0,0 +1,52 @@
+let initialText = prompt('Введите урывок текста');
+
+
+function searchVowels(initialText) { 
+    let count = 0;
+    const vowels = ['a', 'e', 'i', 'o', 'u', 'y']; // проверочный массив гласных 
+    
+    for (let char of initialText.toLowerCase()) { // берем символ из строки и приводим к строчным буквам что бы не вносить в массив проверки доплнительно заглавные буквы
```

Comment:
It's better to normalize initial string only one time than during every iteraction.
```suggestion
    const normalizedInitialText = initialText.toLowerCase();
    for (let char of normalizedInitialText) { // берем символ из строки и приводим к строчным буквам что бы не вносить в массив проверки доплнительно заглавные буквы
```

Reply:
Виконав правки